packages/python/plotly/plotly/express/_doc.py

```diff
@@ -11,377 +10,444 @@
+        "DataFrame or array-like or dict",
+        "This argument needs to be passed for column names (and not keyword names) to be used.",
+        "Array-like and dict are tranformed internally to a pandas DataFrame.",
     ],
     x=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the x axis in cartesian coordinates.",
         "For horizontal `histogram`s, these values are used as inputs to `histfunc`.",
     ],
     y=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the y axis in cartesian coordinates.",
         "For vertical `histogram`s, these values are used as inputs to `histfunc`.",
     ],
     z=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the z axis in cartesian coordinates.",
         "For `density_heatmap` and `density_contour` these values are used as the inputs to `histfunc`.",
     ],
     a=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the a axis in ternary coordinates.",
     ],
     b=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the b axis in ternary coordinates.",
     ],
     c=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the c axis in ternary coordinates.",
     ],
     r=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the radial axis in polar coordinates.",
     ],
     theta=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks along the angular axis in polar coordinates.",
     ],
     lat=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks according to latitude on a map.",
     ],
     lon=[
-        colref,
+        colref_type,
+        colref_desc,
         "Values from this column or array_like are used to position marks according to longitude on a map.",
     ],
     locations=[
-        colref,
-        "Values from this column or array_like are be interpreted according to `locationmode` and mapped to longitude/latitude.",
+        colref_type,
+        colref_desc,
+        "Values from this column or array_like are to be interpreted according to `locationmode` and mapped to longitude/latitude.",
     ],
     dimensions=[
-        "(list of strings, names of columns in `data_frame`)",
-        "Columns to be used in multidimensional visualization.",
+        "list of str",
```

Comment:
I think this is a `colref_list_type` no? @emmanuelle thoughts?

Reply:
yes, correct, we forgot to update this one. @joelostblom would you mind making this change? Otherwise I can push a commit to your branch if you don't have the time. And then we'll merge :-)

Reply:
@emmanuelle Updated!